client/components/Overview/components/ImageGalleryExpand/ImageGalleryExpandedZoom.jsx

```diff
@@ -0,0 +1,16 @@
+/* eslint-disable jsx-a11y/no-static-element-interactions */
+/* eslint-disable jsx-a11y/click-events-have-key-events */
+import React from 'react';
+import '../../styles/ImageGalleryStyles.css';
+
+const ImageGalleryExpandedZoom = ({ currentStyle, currentImageIndex, setIsZoom }) => (
```

Comment:
Does this mean that you figured out how to zoom in? 👏

Reply:
I figured out how to do the modal within the modal. Modalception